app/timedatectl/userparameters/timedatectl.conf

```diff
@@ -0,0 +1,9 @@
+#
+# "NTP enabled" yes/no value from timedatectl output
+#
+UserParameter=rabe.timedatectl.ntp.enabled,/bin/timedatectl --no-pager status | /bin/awk -F ': ' '/NTP enabled:/ { if ($2 == "yes") { print 1; } else { print 0; } }'
+
+#
+# "NTP synchonized" yes/no value from timedatectl output
+#
+UserParameter=rabe.timedatectl.ntp.synchronized,/bin/timedatectl --no-pager status | /bin/awk -F ': ' '/NTP synchronized:/ { if ($2 == "yes") { print 1; } else { print 0; } }'
```

Comment:
To prevent awk from matching possible multiple lines here, you could write the regex in a more restrictive way:
* `[...] /bin/awk -F ': ' '/^[[:space:]]*NTP enabled:/ { if [...]'`
* `[...] /bin/awk -F ': ' '/^[[:space:]]*NTP synchronized:/ { if [...]'`

Reply:
Also note, that the output of `timedatectl status` will change in newer versions and won't work for these `UserParameter`.
Unfortunately, the current version on CentOS / RHEL don't support the [`timedatectl show`](https://www.freedesktop.org/software/systemd/man/timedatectl.html#show) command yet. Maybe add a comment to this script or the README , so that users with newer `timedatectl` versions are warned. The new way to achieve the same result would be to use the more machine friendly `timedatectl show --property=NTPSynchronized --value` command.

Reply:
I adjusted the Regex as per your recommendation, I also added support for modern timedatectl in the enabled regex.